README.md

```diff
@@ -277,27 +277,30 @@ pod "etcd-cluster-with-backup-0001" deleted
 
 Now quorum is lost. The etcd operator will start to recover the cluster by:
 - Creating a new seed member to recover from the backup
-- Add the specified number of members into the seed cluster
+- Add more members until the size reaches to specified number
 
 ```
 $ kubectl get pods
 NAME                                         READY     STATUS     RESTARTS   AGE
-etcd-cluster-with-backup-0002                0/1       Init:0/2   0          11s
+etcd-cluster-with-backup-0003                0/1       Init:0/2   0          11s
 etcd-cluster-with-backup-backup-tool-e9gkv   1/1       Running    0          18m
 ...
 $ kubectl get pods
 NAME                                         READY     STATUS    RESTARTS   AGE
-etcd-cluster-with-backup-0002                1/1       Running   0          3m
 etcd-cluster-with-backup-0003                1/1       Running   0          3m
 etcd-cluster-with-backup-0004                1/1       Running   0          3m
+etcd-cluster-with-backup-0005                1/1       Running   0          3m
 etcd-cluster-with-backup-backup-tool-e9gkv   1/1       Running   0          22m
 ```
 
 Note: Sometimes member recovery can fail because of a race caused by a delay in pod deletion. The process will be retried in this event.
 
 ## Upgrade an etcd cluster
 
-Clean up any existing etcd cluster, but keep the etcd operator running.
+Continued from last example, clean up existing cluster:
```

Comment:
It's been consistently continuous in this readme.
Let me do it in another PR to make each section independent.